gapminder_analytics.ipynb

```diff
@@ -0,0 +1,1523 @@
+       "      <td>2002</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1703</th>\n",
+       "      <td>Zimbabwe</td>\n",
+       "      <td>Africa</td>\n",
+       "      <td>2007</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "       country continent  year\n",
+       "1694  Zimbabwe    Africa  1962\n",
+       "1695  Zimbabwe    Africa  1967\n",
+       "1696  Zimbabwe    Africa  1972\n",
+       "1697  Zimbabwe    Africa  1977\n",
+       "1698  Zimbabwe    Africa  1982\n",
+       "1699  Zimbabwe    Africa  1987\n",
+       "1700  Zimbabwe    Africa  1992\n",
+       "1701  Zimbabwe    Africa  1997\n",
+       "1702  Zimbabwe    Africa  2002\n",
+       "1703  Zimbabwe    Africa  2007"
+      ]
+     },
+     "execution_count": 17,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "subset.tail(10)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "small_range = list(range(5))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 23,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "country      Afghanistan\n",
+       "continent           Asia\n",
+       "year                1952\n",
+       "lifeExp           28.801\n",
+       "pop              8425333\n",
+       "gdpPercap        779.445\n",
+       "Name: 0, dtype: object"
+      ]
+     },
+     "execution_count": 23,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# We can use .loc method on the DF to subset rows based on the index label\n",
+    "subset = df.loc[0] # Get the first row\n",
+    "subset"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 26,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "country      Bangladesh\n",
+       "continent          Asia\n",
+       "year               1967\n",
+       "lifeExp          43.453\n",
+       "pop            62821884\n",
+       "gdpPercap       721.186\n",
+       "Name: 99, dtype: object"
+      ]
+     },
+     "execution_count": 26,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Get the 100th row\n",
+    "df.loc[99]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 30,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "country      Zimbabwe\n",
+       "continent      Africa\n",
+       "year             2007\n",
+       "lifeExp        43.487\n",
+       "pop          12311143\n",
+       "gdpPercap     469.709\n",
+       "Name: 1703, dtype: object"
+      ]
+     },
+     "execution_count": 30,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "df.iloc[-1]"
```

Comment:
shows all the information from the indexed line starting from the end of the list